1839 - Document ignore_default field for account_lines API method (#2970)
XRPLF/xrpl-dev-portal: docs/references/http-websocket-apis/public-api-methods/account-methods/account_lines.md

The request accepts the following parameters:

| `Field`          | Type                 | Description    |
|:-----------------|:---------------------|:---------------|
| `account`        | String - [Address][] | Look up trust lines connected to this account. |

> | `ignore_default` | Boolean              | _(Optional)_ If true, suppress the output of incoming trust lines in the default state. Defaults to false. |

Reviewer: Nit: slight formatting/wording edits.

```suggestion
| `ignore_default` | Boolean              | _(Optional)_ If `true`, suppress the output of incoming trust lines in the default state. The default is `false`. |
```
